src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBaseKernels.hpp

```diff
@@ -540,18 +539,18 @@ struct SolutionCheckKernel
   template< typename POLICY >
   static localIndex launch( arrayView1d< real64 const > const & localSolution,
                             globalIndex const rankOffset,
-                            arrayView1d< globalIndex const > const & presDofNumber,
+                            arrayView1d< globalIndex const > const & dofNumber,
                             arrayView1d< integer const > const & ghostRank,
                             arrayView1d< real64 const > const & pres,
                             real64 const scalingFactor )
   {
     RAJA::ReduceMin< ReducePolicy< POLICY >, localIndex > minVal( 1 );
 
-    forAll< POLICY >( presDofNumber.size(), [=] GEOSX_HOST_DEVICE ( localIndex const ei )
+    forAll< POLICY >( dofNumber.size(), [=] GEOSX_HOST_DEVICE ( localIndex const ei )
     {
-      if( ghostRank[ei] < 0 && presDofNumber[ei] >= 0 )
+      if( ghostRank[ei] < 0 && dofNumber[ei] >= 0 )
```

Comment:
> why this change? is it because this now is used for both pressure and temperature?

This function is still used for pressure only, but I think it will be extended to include temperature. So I change the name of this parameter.